lib/main.js

```diff
@@ -132,6 +132,17 @@ async function main (args = null) {
     await server.close();
     throw err;
   }
+
+  process.on('SIGINT', function() {
+    logger.info(`Received SIGINT - shutting down`);
+    server.close();
```

Comment:
we `await server.close()` earlier, so we should probably convert this into an async function for consistency.